helm/default-apps-cloud-director/values.schema.json

```diff
@@ -3,66 +3,357 @@
     "type": "object",
     "properties": {
         "apps": {
-            "additionalProperties": {
```

Comment:
why do we change `additionalProperties`?

Reply:
I ran helm schema-gen so it removed all of those. Do you manage the schéma by hand?

Reply:
yeah :(